setup.py

```diff
@@ -426,6 +426,17 @@ def make_deb():
     os.mkdir("./xpra-html5")
     shutil.copytree("./packaging/debian", "./xpra-html5/DEBIAN")
     install_html5("./xpra-html5/usr/share/xpra/www/", "uglifyjs")
+    # Move and symlink default-settings.txt
+    os.makedirs("./xpra-html5/etc/xpra/html5-client", exist_ok=True)
+    default_settings_files = { "default-settings.txt", "default-settings.txt.gz", "default-settings.txt.br" }
+    for filename in default_settings_files:
```

Comment:
I don't like having 3 copies of what is effectively the same file in `/etc`
We should just prevent this file from ever being minified instead.

Reply:
Yes, I will fix it!